### ISSUE TITLE
'poetry env info --path' returns empty output

### ISSUE THREAD

**Issue by ovstuckrad** [python-poetry/poetry#7396]
- **1.3.2**: 
- **3.11.1**: 
- **macOS 13.1 (Apple Silicone/M1)**: 

- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) stable Poetry version, installed using a recommended method.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have consulted the [FAQ](https://python-poetry.org/docs/faq/) and [blog](https://python-poetry.org/blog/) for any relevant entries or release notes.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option) and have included the output below.

## Issue

When executing `poetry env info --path`, I only get an empty newline as return. When running `poetry env info`, the path is shown as "NA".

I would like to use this command to source the environment in my bashrc file dynamically.

```
❯ poetry env info

Virtualenv
Python:         3.11.1
Implementation: CPython
Path:           NA
Executable:     NA

System
Platform:   darwin
OS:         posix
Python:     3.11.1
Path:       /opt/homebrew/Cellar/poetry/1.3.2/libexec
Executable: /opt/homebrew/Cellar/poetry/1.3.2/libexec/bin/python3.11
```

**Comment by dimbleby** [comment]
are you in a virtual environment?

**Comment by dimbleby** [comment]
> I would like to use this command to source the environment in my bashrc file dynamically.

sounds like you have things back to front.  `poetry env info` tells you about the active environmont: that is, you need to activate the environment _before_ running this command
```
$ poetry env info --help

Description:
  Displays information about the current environment.
```

**Comment by finswimmer** [comment]
`poetry env info` shows you the paths to the venv as soon as one exists. (It doesn't have to be activated). So I guess you don't have created a venv yet?

**Comment by ovstuckrad** [comment]
Hi, sorry for the late reply and thanks for your help.

I did not have an external venv. I created one manually and activated it. Running `poetry install` in the external venv still created a new virtual env (under the Caches/pypoetry path) and did not show a path from the `poetry env info --path`.

After running `poetry env use system` once it is now working as expected - I can switch to my externally created venv, use poetry to install in it and `poetry env info --path` is showing the path to the venv.

So I think you were correct, I didn't have an external venv. After creating it, I also needed to tell poetry to use the system's python because otherwise it didn't know about the external venv.

A bit confusing but probably not a bug.
